app/posts/[identifier]/page.tsx

```diff
@@ -3,6 +3,12 @@ import { getMumblePostAction } from '@/utils/helpers/posts/getMumblePost';
 import React from 'react';
 import { PostFull } from '@/components/post-full/PostFull';
 import { auth } from '@/app/api/auth/[...nextauth]/auth';
+import type { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'Mumble | Post Full',
+  description: 'Mumbles/Likes/Followers/Following/Suggestions',
```

Comment:
The description can be defined in app/layout.tsx and all children will inherit the description. So, you don't need to define it on each page.tsx/layout.tsx.

Reply:
Maybe we can add then personalised descriptions for all pages.